Tests/OpenChat.API.UnitTests/UsersControllerTests.cs

```diff
@@ -1,41 +1,75 @@
+using System;
 using System.Threading.Tasks;
 using FluentAssertions;
 using Microsoft.AspNetCore.Mvc;
 using Moq;
 using OpenChat.API.Controllers;
+using OpenChat.API.Models;
 using OpenChat.Application.Users;
 using Xunit;
 
 namespace OpenChat.API.UnitTests
 {
     public class UsersControllerTests
     {
+        // [Fact]
```

Comment:
This commented test can be deleted. Or it can be left for reference on the size of the steps when doing TDD and deciding on what the interaction between the sut and it's collaborator should be.